bundles/org.openhab.binding.xmppclient/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.xmppclient</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: XMPPClient Binding</name>
+  
+  <dependencies>
+        <dependency>
+            <groupId>org.igniterealtime.smack</groupId>
+            <artifactId>smack-java7</artifactId>
+            <version>4.3.1</version>
```

Comment:
The latest Smack version at the time of writing this is 4.3.3  (ETA for 4.3.4 is in some weeks). I also suggest creating a single 'smackVersion' variable and referencing that variable in the dependency declarations.

Reply:
Unfortunately the new version is no longer an OSGi bundle, they have removed that support.

Reply:
I have removed the (deprecated) OSGi gradle-plugin in Smack 4.4, not in Smack 4.3.

I always wonder if Smack declaring the OSGi specific metadata in its Manifest files is a must to be used in OSGi projects. Wasn't there a way to just import non OSGi artifacts and threat them like they are part of the importing bundle?

The reason for https://github.com/igniterealtime/Smack/commit/d06f533bb975f41a5e86c990e1bb830b7e7dc923 is simply that the gradle plugin Smack used to become an OSGi bundle is deprecated and the recommended alternative plugin did not work. I have myself no need for OSGi, but for obvious reasons do not want to carry around a deprecated plugin. Of course that does not mean that I won't be thankful for any help or contribution in this regard.